src/main/java/net/sf/jabref/logic/util/OptionalUtil.java

```diff
@@ -0,0 +1,26 @@
+package net.sf.jabref.logic.util;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.bouncycastle.asn1.ua.DSTU4145NamedCurves.params;
+
+public class OptionalUtil {
```

Comment:
Looks good. I wonder, why there is no other similar class already existing in the OSS universe. Nevertheless, let's go forward!